wix-inbox-v2/wix-inbox-v2/Messages/guides/messages-intro.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Introduction
+---
+
+# Introduction
+
+&nbsp;
+
+<div style="background-color: #FEF1D1; padding: 18px 24px; border-radius: 6px; border: 1px solid #FDB10C; box-sizing: border-box; display: inline-block">
+    <b>Developer Preview</b>
+    <br/>
+    <span>APIs in <a href="https://www.wix.com/velo/reference/api-overview/developer-preview">Developer Preview</a> are subject to change and are not intended for use in production.<br/><a href="mailto:velo-preview-feedback@wix.com">Send us</a> your suggestions for improving this API. Your feedback is valuable to us.</span>
+</div>
+
+&nbsp;
+
+<!-- > __Note:__
+> This module is [universal](/api-overview/api-versions#universal-modules).
+> Functions in this module can run on both the backend and frontend,
+> unless specified otherwise. -->
+
+The Messages API allows you to send messages to participants and read previously sent messages. 
```

Comment:
How is this different from the Conversations API?- Since you can retrieve conversations and create new conversations there too. Maybe you can clarify when you would choose to use one over the other?